### ISSUE TITLE
[Bug]: Account has been reverted to the free version

### ISSUE THREAD

**Issue by Muzzlebean** [MelvorIdle/melvoridle.github.io#2619]
### Describe the bug

I logged in this morning and my full account (I made an account before the premium change) has been downgraded to the free version. I have logged out and deleted app, but still no change. Hope you can help! 

### Save Link

https://melvoridle.com/save/qfn1Kc6V

### Which platforms are you experiencing this bug on?

- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari
- [ ] Mobile App (iOS)
- [X] Mobile App (Android)
- [ ] Steam
- [ ] Other (Please Specify)

### Which version of the game are you experiencing this bug on?

v1.0.5

### Game Subversion

?2132

### Are you using any scripts or extensions?

No

**Comment by Shroomicus** [comment]
I'm having this same issue. Tested on browser, and it registers correctly that my account is old enough, but on mobile no such luck for the same save. Both Melvor and Playfab connections are green.

**Comment by Muzzlebean** [comment]
Yeah I just tried browser and it registers the full version, but still no luck on app.

**Comment by Maidstone2000** [comment]
Web Browser version changed for me to the free version, so has the Android App.

**Comment by Syllilith** [comment]
App Version has Reverted for me too
